intermediate_source/bettertransformer_tutorial.rst

```diff
@@ -0,0 +1,250 @@
+Fast Transformer Inference with Better Transformer
+===============================================================
+
+**Author**: `Michael Gschwind <https://github.com/mikekgfb>`__
+
+This tutorial introduces Better Transformer (BT) as part of the PyTorch 1.12 release. 
+In this tutorial, we show how to use Better Transformer for production 
+inference with torchtext. 
+
+The example uses a Google Colab notebook to demonstrate Better Transformer setup
+and usage for simplicity. Better Transformer is a production ready fastpath to
+accelerate deployment of Transformer models with high performance on CPU and GPU.
+The fastpath feature works transparently for models based either directly on 
+PyTorch core nn.module or with torchtext.  
+
+Models which can be accelerated by Better Transformer fastpath execution are those
+using the following PyTorch core torch.nn.module with the TransformerEncoder, 
+TransformerEncoderLayer, and MultiHeadAttention models.  In addition, torchtext has 
+been updated to use the core library modules to benefit from the acceleration.
+(Additional modules may be enabled with fastpath execution in the future.)
+
+Better Transformer offers two types of acceleration:
+
+* Native multihead attention implementation for CPU and GPU to improve overall execution efficiency.  
+* Exploiting sparsity in NLP inference.  Because of variabl input lengths, input
+  tokens may contain a large number of padding tokens for which processing may be
+  skipped, delivering significant speedups.
+
+Fastpath execution is subject to some criteria. Most importantly, the model 
+must be executed in inference mode and operate on input tensors that do not collect 
+gradient tape information (e.g., running with torch.no_grad). 
+
+Better Transformer Features in This Tutorial
+--------------------------------------------
+* Load pre-trained models (pre-1.12 created without Better Transformer)
+* Run and benchmark inference on CPU with and without BT fastpath (native MHA only)
+* Run and benchmark inference on (configurable) DEVICE with and without BT fastpath (native MHA only)
+* Enable sparsity support
+* Run and benchmark inference on (configurable) DEVICE with and without BT fastpath (native MHA + sparsity)
+
+Additional Information
+-----------------------
+Additional iinformation about Better Transformer may be found in the PyTorch.Org blog  
```

Comment:
```suggestion
Additional information about Better Transformer may be found in the pytorch.org blog  
```
pytorch.org blog  - can we make this a link to the blog?

Reply:
I think the next line has that link?